routers/web/user/profile.go

```diff
@@ -91,6 +92,50 @@ func Profile(ctx *context.Context) {
 		ctx.Data["RenderedDescription"] = content
 	}
 
+	repo, err := repo_model.GetRepositoryByName(ctx.ContextUser.ID, ".profile")
+	if err == nil && !repo.IsEmpty {
+		gitRepo, err := git.OpenRepository(ctx, repo.RepoPath())
+		if err != nil {
+			ctx.ServerError("OpenRepository", err)
+			return
+		}
+		defer gitRepo.Close()
+		defaultBranch := repo.DefaultBranch
+		commitID, err := gitRepo.GetBranchCommitID(defaultBranch)
+		if err != nil {
+			ctx.ServerError("GetBranchCommitID", err)
+			return
+		}
+		commit, err := gitRepo.GetCommit(commitID)
+		if err != nil {
+			ctx.ServerError("GetCommit", err)
+			return
+		}
+		tree, err := commit.SubTree("")
+
+		if err != nil || tree == nil {
+			ctx.ServerError("SubTree", err)
+			return
+		}
+		blob, err := tree.GetBlobByPath("README.md")
```

Comment:
```suggestion
		blob, err := commit.GetBlobByPath("README.md")
```

https://github.com/go-gitea/gitea/blob/2ec2d065316a488c4748061fe7b19f0b57f3546f/modules/git/tree.go#L21-L24
`commit.SubTree("")` will return itself, so maybe it is unnecessary to use `SubTree` here